tailwindcss/import_map.json

```diff
@@ -0,0 +1,6 @@
+{
+  "imports": {
+    "lume/": "https://deno.land/x/lume@v1.11.2/",
+    "dbin/": "https://deno.land/x/dbin@v0.1.1/"
```

Comment:
Using import maps for the dependencies force to the users that want to use this plugin to update their import maps (as you had to do in the demo)